### PR TITLE
zh-translation:/docs/tasks/security/mutual-tls/index.md

### PR DESCRIPTION
**Pull request by JasonRD**
ref: https://github.com/servicemesher/istio-official-translation/issues/1075

### CODE REVIEW

File: content/zh/docs/tasks/security/mutual-tls/index.md [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
-* You are familiar with using authentication policy to enable mutual TLS.
-* Istio runs on Kubernetes with global mutual TLS enabled. You can follow our [instructions to install Istio](/docs/setup/).
-If you already have Istio installed, you can add or modify authentication policies and destination rules to enable mutual TLS as described in this [task](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls).
-* You have deployed the [httpbin]({{< github_tree >}}/samples/httpbin) and [sleep]({{< github_tree >}}/samples/sleep) with Envoy sidecar in the `default` namespace. For example, below is the command to deploy those services with [manual sidecar injection](/docs/setup/additional-setup/sidecar-injection/#manual-sidecar-injection):
+* 您已经完成[认证策略](/zh/docs/tasks/security/authentication/authn-policy/) 任务.
+* 您熟悉如何通过认证策略开启双向 TLS。
+* Istio 在 Kubernetes 上运行，并且开启全局双向 TLS。可以按照 [Istio 安装说明文档](/zh/docs/setup/))。
+如果已经安装了 Istio 的安装，可以根据[为所有服务启用双向 TLS 认证](/zh/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls) 任务中说明，通过增加或者修改认证策略和目的规则来开启双向 TLS。
```
Comment (rootsongjc) [inline content/zh/docs/tasks/security/mutual-tls/index.md]: 安装了 Istio 的安装？

**Comment by rootsongjc** [comment]
Request for a reviewer.

File: content/zh/docs/tasks/security/mutual-tls/index.md (hunk 1 above)
Comment (JasonRD) [reply content/zh/docs/tasks/security/mutual-tls/index.md]: 3ks, fix it

File: .spelling [hunk 2]
```diff
@@ -346,6 +346,7 @@ multicloud
 Multicluster
 multicluster
 mutual-tls
+tls
```
Comment (rootsongjc) [inline .spelling]: Do not modify this file.

Comment (JasonRD) [reply .spelling]: ok, rollback it.

**Pull request merged**